frontend/src/views/Project/CertificatesPage/components/CertificatesTab/components/CertificatesTable.utils.ts

```diff
@@ -0,0 +1,28 @@
+import ms from "ms";
+
+export const getCertValidUntilBadgeDetails = (notAfter: string) => {
```

Comment:
what do you think about adding this badge thing to the collection certs view as well?
<img width="1224" alt="image" src="https://github.com/user-attachments/assets/f5eea93f-f91c-44f5-b6c7-624cc5545981">

Reply:
Will do this in a separate PR!